guides/common/modules/con_smartproxy-overview.adoc

```diff
@@ -0,0 +1,12 @@
+[id="{SmartProxy}-Overview_{context}"]
+= {SmartProxy} overview
+
+With {SmartProxyServers}, you can extend the reach and scalability of your {Project} deployment.
+{SmartProxyServers} provide the following functionalities in a {ProjectName} deployment:
+
+* Mirroring content from {ProjectServer} to establish content sources in various geographical locations.
+By assigning a host system to a {SmartProxyServer}, you can configure this host to pull content and configuration from the {SmartProxy} in their location instead of from the central {ProjectServer}.
+* Running localized services to discover, provision, control, and configure hosts.
+
+You can specify the exact subset of content that {SmartProxyServer} makes available to hosts by using content views.
```

Comment:
Maybe reword to "You can use content views to specify the exact subset of content that {SmartProxyServers} provide to hosts"?

Reply:
Applied.